DirectoryListing/Entities/Individual.swift

```diff
@@ -47,60 +47,82 @@ class Individual: BaseEntity {
         affiliation <- map["affiliation"]
     }
     
-    func friendlyBirthdate() -> String? {
-        return self.birthdate?.friendlyDate()
+    func friendlyBirthdate() -> String {
+        guard let tempDate = self.birthdate.friendlyDate() else {
+            return ""
```

Comment:
I think it's better to return an optional rather than the empty string (or other sentinel values) from helper methods like this because then whatever calls the function can decide on the right thing to do for nil values (sometimes putting the empty string in it's place isn't the right thing to do everywhere).